### DISCUSSION TITLE
connect ECONNREFUSED 127.0.0.1:443

### DISCUSSION THREAD

**Discussion by litt1e-p** [community/community#17263] [Copilot Conversations]
Copilot ativate failed with an output: **GitHub Copilot could not connect to server. Extension activation failed: "connect ECONNREFUSED 127.0.0.1:443"**

copilot version: v1.23.5969
vs code :
Version: 1.67.2 (user setup)
Commit: c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5
Date: 2022-05-17T18:15:52.058Z
Electron: 17.4.1
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Windows_NT x64 10.0.19041

**Comment by litt1e-p** [comment]
ignore, it's fixed

**Reply by Yunnathai** [reply]
how fixed . same with you.

**Reply by XiaoYingYo** [reply]
> 忽略，已修复

How to solve it please?

**Reply by litt1e-p** [reply]
change the dns to 8.8.8.8

**Reply by XiaoYingYo** [reply]
> 8.8.8.8

Thank you very much, I used this method and solved it

**Reply by RSS1102** [reply]
Thank you，me too. Fixed 。
